client/src/thin_client.rs

```diff
@@ -648,23 +638,15 @@ impl<C: 'static + TpuConnection> AsyncClient for ThinClient<C> {
     }
 }
 
-pub fn create_client(
-    (rpc, tpu): (SocketAddr, SocketAddr),
-    range: (u16, u16),
-) -> ThinClient<UdpTpuConnection> {
-    let (_, transactions_socket) =
-        solana_net_utils::bind_in_range(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), range).unwrap();
-    ThinClient::<UdpTpuConnection>::new(rpc, tpu, transactions_socket)
+pub fn create_client((rpc, tpu): (SocketAddr, SocketAddr)) -> ThinClient {
```

Comment:
Why not use individual parameters which is more intuitive as opposed to a tuple?

Reply:
please do this

Reply:
Hmm... it seems a lot of places return (rpc, tpu) tuples, and all of these places would need to be refactored, or we would need to extract out the components from all those places. Seems a little too invasive for a change like this...

Reply:
Right. I missed that this was retaining old dumb behavior rather than adding new dumb behavior.  Let's add an issue with suggestion to refactor and punt for now